check-formatting.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Pre-commit hook for StartupOS
+# Run it before committing to verify whether all files 
+# are properly formatted so you won't fail early on review
+# Either do it manually or by creating a symlink
+# To do this, execute, from repo root
+# ln -s $(pwd)/pre-commit.sh $(pwd)/.git/hooks/pre-commit
```

Comment:
How about instead of precommit, we add it to `aa fix`?

Reply:
@oferb yeah, let's remove precommit mentions from both files (`check-formatting.sh` and `pre-commit.sh`)